components/drop-zone/style.scss

```diff
@@ -0,0 +1,52 @@
+.components-drop-zone {
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	margin: 15px;
+	z-index: z-index( '.components-drop-zone' );
+	visibility: hidden;
+	opacity: 0;
+	transition: 0.3s opacity, 0.3s background-color, 0s visibility 0.3s;
+	border: 2px solid $blue-wordpress;
+	border-radius: 2px;
+	background-color: rgba( $blue-medium-500, 0.8 );
+
+	&.is-active {
+		opacity: 1;
+		visibility: visible;
+		transition: 0.3s opacity, 0.3s background-color;
+	}
+
+	&.is-dragging-over-element {
```

Comment:
>I am seeing the area highlighted even when dragging outside of the image block.

Hmm, there's supposed to be two levels of highlight: One for when dragging over the page to indicate where drop zones are, then another when over the droppable area. Seems to not be working at the moment.